src/backend/config/KsnipConfig.cpp

```diff
@@ -223,6 +223,20 @@ void KsnipConfig::setApplicationStyle(const QString &style)
 	saveValue(KsnipConfigOptions::applicationStyleString(), style);
 }
 
+int KsnipConfig::trayLeftClickAction() const
```

Comment:
I think the name should be "defaultTrayIconAction", the user might click with something different.